client/src/main/java/org/apache/rocketmq/client/ClientConfig.java

```diff
@@ -340,6 +341,22 @@ public void setEnableStreamRequestType(boolean enableStreamRequestType) {
         this.enableStreamRequestType = enableStreamRequestType;
     }
 
+    public void setZoneName(String zoneName) {
```

Comment:
It is very difficult to let customers to upgrade the client version. If this zone information can be obtained through ip or other environmental information, it will be more friendly